src/stories/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import './button.css';
+
+export interface ButtonProps {
+  primary?: boolean;
+  backgroundColor?: string;
+  size?: 'small' | 'medium' | 'large';
+  label: string;
+  onClick?: () => void;
+}
+
+export const Button = ({
```

Comment:
Do you actually need this separate `Button` component and separate story for it?
You could use your Button component, which is actually used in your project